pkg/windows/windows.go

```diff
@@ -1,16 +1,22 @@
 package windows
 
 import (
+	"context"
 	"fmt"
+	"net/url"
 	"path/filepath"
 	"strings"
 	"time"
 
 	"github.com/go-logr/logr"
+	config "github.com/openshift/api/config/v1"
```

Comment:
found several ways to name this import reference in the code base, for example [oconfig](https://github.com/openshift/windows-machine-config-operator/blob/c2ec776ed0474386311f400b20c2cf07d926c990/controllers/windowsmachine_controller.go#L9) and [config](https://github.com/openshift/windows-machine-config-operator/blob/26fa97a5f3822f0f3eb1787707e8996b02a984ce/test/e2e/create_test.go#L11). 

What would be the consensus here?